balancer-js/src/modules/pools/pool-types/concerns/types.ts

```diff
@@ -16,6 +16,15 @@ export interface SpotPriceConcern {
   ) => string;
 }
 
+export interface PriceImpactConcern {
+  bptZeroPriceImpact: (pool: Pool, tokenAmounts: bigint[]) => bigint;
```

Comment:
@johngrantuk - I know we were avoiding BigNumber as input/output, but I'm not sure about bigint. Should we keep everything as `string` or it's ok to have `bigint` as well?

Reply:
Yeah, lets try and stick to strings as inputs/outputs for external/user facing functions.

Reply:
We discussed with Bruno about this because the method bptZeroPriceImpact doesn't seem to be user facing. Changing that type would imply adding two unnecessary conversions.

Reply:
Apologies, I wasn't sure if that was user facing. If its not then let leave it as bigint as you suggest.